src/model/exceptions/already-exist-exception.ts

```diff
@@ -0,0 +1,8 @@
+import { Exception } from './exception'
+import { ErrorCode } from '../enums/error-code'
+
+export class AlreadyExistException extends Exception {
+  constructor(message?: string) {
```

Comment:
I would prefer that we make the message required for all exceptions. It forces us to write (hopefully) useful information  about what goes wrong in specific scenarios.

Reply:
Updated